extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -690,6 +706,15 @@ export class AmpStoryPage extends AMP.BaseElement {
     return isMediaDisplayed(ampEl, resource);
   }
 
+  /**
+   * @return {!Promise<boolean>}
+   * @private
+   */
+  isAutoplaySupported_() {
+    VideoUtils.resetIsAutoplaySupported();
+    return VideoUtils.isAutoplaySupported(this.win, getMode(this.win).lite);
```

Comment:
What's `lite` for?

Reply:
From the comments, it's an `amp_lite` mode that can be enabled by the viewer to use less bandwidth. The only difference I've been able to find after a 10 seconds search is that it doesn't autoplay videos. No idea what viewers/platforms enable this though...